superdesk/commands/index_from_mongo.py

```diff
@@ -22,20 +24,29 @@ class IndexFromMongo(superdesk.Command):
     """
 
     option_list = [
-        superdesk.Option('--from', '-f', dest='mongo_collection_name', required=True),
+        superdesk.Option('--from', '-f', dest='mongo_collection_name'),
         superdesk.Option('--page-size', '-p', dest='page_size')
     ]
     default_page_size = 500
 
     def run(self, mongo_collection_name, page_size):
-        for items in self.get_mongo_items(mongo_collection_name, page_size):
+        if not mongo_collection_name:
+            app.data.init_elastic(app)
+            resources = app.data.get_elastic_resources()
+            for resource in resources:
+                self._copy_resource(resource, page_size)
+        else:
+            self._copy_resource(mongo_collection_name, page_size)
```

Comment:
It would be good have `mongo_collection_name` as required parameter. Maybe specify value=all  to initialize all indexes. Basically want to avoid someone run re-index for all collection.